src/applications/disability-benefits/781/pages/individualsInvolvedChoice.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { individualsInvolvedTitle } from '../helpers';
+
+const individualsInvolvedChoiceDescription = () => (
+  <div>
```

Comment:
You can remove this `<div>`. Since the function already only returns 1 .jsx node, you don't need the wrapping element.

Reply:
Awww, yep!